src/main/java/net/glowstone/util/Wearable.java

```diff
@@ -0,0 +1,120 @@
+package net.glowstone.util;
+
+import org.bukkit.enchantments.EnchantmentTarget;
+import org.bukkit.Material;
+import java.util.Map;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.inventory.ItemStack;
+import java.util.HashMap;
+import java.util.Set;
+
+public final class Wearable {
+    
+    private static Map<Material, WearableReturn> map;
+    private static boolean intalized = false;
+    
+    private class WearableReturn {
+        private EnchantmentTarget wearablePosition;
+        private int armorPoints;
+        
+        public WearableReturn(EnchantmentTarget position, int points) {
+            this.wearablePosition = position;
+            this.armorPoints = points;
+        }
+        public EnchantmentTarget wearPosition() {
+            return wearablePosition;
+        }
+        public int armorPoints() {
+            return armorPoints;
+        }
+    }
+    
+    protected Map<Material, WearableReturn> createMap() {
+        Map<Material, WearableReturn> map2 = new HashMap<>();
+        map2.put(Material.LEATHER_BOOTS, new WearableReturn(EnchantmentTarget.ARMOR_FEET, 1));
```

Comment:
Don't use `EnchantmentTarget` for specifying slot, only for determining it. Slot should be specified with `EquipmentSlot` (if needed) and it is not needed to define armor to be put in a certain slot, as that can be checked with the `includes(Material)` method in `EnchantmentTarget`. For example, `EnchantmentTarget.ARMOR_HEAD.includes(Material.GOLD_HELMET)` would return true.